presets/wikimedia.json

```diff
@@ -65,5 +65,6 @@
     "disallowNewlineBeforeBlockStatements": true,
     "validateLineBreaks": "LF",
     "validateQuoteMarks": "'",
-    "validateIndentation": "\t"
+    "validateIndentation": "\t",
+    "requireSpacesInForStatement": true
```

Comment:
@markelog Ah, I see. Nice! I still feel like this should maybe be generalised in some way, but sure, feel free to add this to the wikimedia preset.